### PR TITLE
Global css

### PR DESCRIPTION
**Pull request by Mussuranae**
CSS Review of forms

### CODE REVIEW

File: libs/movie/src/lib/movie/form/festival-prizes/festival-prizes.component.html [hunk 1]
```diff
@@ -3,13 +3,16 @@
   <h3> Festival Prizes </h3>
   <div formArrayName="prizes" *ngFor="let credit of festivalprizes.prizes.controls; let i = index">
     <div [formGroupName]="i" class="full-width">
-      <mat-form-field class="small-margin">
+      <mat-form-field appearance="outline" class="small-margin">
```
Comment (Hugo-Hontans) [inline libs/movie/src/lib/movie/form/festival-prizes/festival-prizes.component.html]: Do we really need the ``class="small-margin`` ?

File: libs/movie/src/lib/movie/form/main/main.component.html [hunk 2]
```diff
@@ -1,13 +1,15 @@
 <form page-id="movie-form-main" [formGroup]="main">
   <!-- INTERNAL REF -->
-  <mat-form-field class="full-width">
+  <mat-form-field appearance="outline" class="full-width">
```
Comment (Hugo-Hontans) [inline libs/movie/src/lib/movie/form/main/main.component.html]: Same, do we really need the ``class=full-width`` ?

Comment (Mussuranae) [reply libs/movie/src/lib/movie/form/main/main.component.html]: It is in the parent component, if we delete it, inputs are all very closed and little. We will refactor this form later I think.

File: libs/movie/src/lib/movie/form/festival-prizes/festival-prizes.component.html (hunk 1 above)
Comment (Mussuranae) [reply libs/movie/src/lib/movie/form/festival-prizes/festival-prizes.component.html]: Yes for the form, if we delete, all inputs are stuck.

**Pull request merged**